src/libraries/Microsoft.Bcl.Memory/tests/Microsoft.Bcl.Memory.Tests.csproj

```
@@ -39,7 +39,7 @@
     </Compile>
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' != '.NETCoreApp'">
+  <ItemGroup Condition="!$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net9.0'))">
```

nit: I don't think that we need this condition at all.

---

Right, good catch, it was needed when there was duplicate code, now `.NET 9` and `netstandard` shares the code
```suggestion
  <ItemGroup>
```